ui/apps/platform/package.json

```diff
@@ -49,6 +50,7 @@
         "lodash": "^4.17.21",
         "mobx": "^6.3.2",
         "mobx-react": "^7.5.3",
+        "monaco-editor": "^0.34.1",
```

Comment:
Had some issues with adding the @patternfly/react-code-editor library. I had to install this one as well to fix the issues

Reply:
Yea, otherwise this happens

![Uploading Screen Shot 2022-10-20 at 10.45.44 AM.png…]()